README.md

```diff
@@ -3,15 +3,15 @@
 #### For Users
 
 * <a href="https://powerofus.force.com" target="_blank">Ask questions or get help</a>
-* <a href="https://github.com/SalesforceFoundation/Cumulus/issues/new" target="_blank">Log a confirmed Issue or Feature Request</a>
+* <a href="https://github.com/SalesforceFoundation/Cumulus/issues/new" target="_blank">Log a confirmed Issue</a> or <a href="https://powerofus.force.com/hub-ideas" target="_blank">Feature Request</a>
```

Comment:
Looks like these were Judi's changes to dev, not sure why they're not in both of these branches.